third_party/libunwindstack/PeCoffInterface.cpp

```diff
@@ -468,6 +468,18 @@ bool PeCoffInterfaceImpl<AddressType>::MapFromRvaToFileOffset(uint64_t rva, uint
   return false;
 }
 
+template <typename AddressType>
+bool PeCoffInterfaceImpl<AddressType>::MapFromFileOffsetToRva(uint64_t file_offset, uint64_t* rva) {
+  for (const Section& section : sections_) {
```

Comment:
So `sections_` are the sections in the Pe/Coff file. So the map info tells us an "object offset". And now we are looking for that section that this offset corresponds to and convert it into relative virtual address. Right?

Reply:
Exactly. (Think of "object offset" just as the offset value of the map. `MapInfo` distinguishes between "offset" and "object offset" for some special cases with ELF files.)